### ISSUE TITLE
Added ru-RU translation

### ISSUE THREAD

**Issue by ashed** [RealRaven2000/FiltaQuilla#100]
I added translation of these resources
filtaquilla@mesquilla.com/locale/ru-RU/prefwindow.dtd
filtaquilla@mesquilla.com/locale/ru-RU/filtaquilla.properties
filtaquilla@mesquilla.com/locale/ru-RU/jsEditor.dtd
filtaquilla@mesquilla.com/locale/ru-RU/bindings.dtd
filtaquilla@mesquilla.com/locale/ru-RU/am-inheritPane.properties
filtaquilla@mesquilla.com/locale/ru-RU/am-inheritPane.dtd

Add it to upstream please.
[ru-RU.zip](https://github.com/RealRaven2000/FiltaQuilla/files/6273314/ru-RU.zip)

**Comment by RealRaven2000** [comment]
Thanks - added to ESR78. Your translation is probably based on a slightly older version of FiltaQuilla (not 3.1) - the last one supports sound. so the following lines / entities are missing:

prefwindow.dtd:
```
<!ENTITY checkTonequillaEnabled.label "Play Sound (Tonequilla)">
```

filtaquilla.properties:
```
filtaquilla.playSound=Play Sound
filtaquilla.tone.select=Select a Sound File…
```

What shall I write for translation credentials in the credits?

Minor layout problems in options dialog caused by accelerators (not included in original strings), which I have yet to solve:
![image](https://user-images.githubusercontent.com/6314674/113909328-77458000-97cf-11eb-9719-f6d641ddddb5.png)

**Comment by ashed** [comment]
Yes, my translation is based on a slightly older version of FiltaQuilla (2.1.2) Thunderbird 68.10.0 Linux

I add missed tranlation in this comment.
---
prefwindow.dtd:

<!ENTITY checkTonequillaEnabled.label "Проиграть звук (Tonequilla)">
---
filtaquilla.properties:

filtaquilla.playSound=Проиграть звук
filtaquilla.tone.select=Выберите звуковой файл…
---

"What shall I write for translation credentials in the credits?"
Zhigunov Igor aka ashed

And how you will fix minor layout problems in options dialog?
Maybe expand place for translated strings?

**Comment by ashed** [comment]
fixed mistype please
filtaquilla.runProgram.select=Выберите программу…

**Comment by raspopov** [comment]
Last f4e691c6534e2b9cc5d18b8a2941f02f1650fb63 update introduces a many errors in the Russian locale, for example now:
```
  "fq.saveMsgAsFile": {
    "message": "Сох"
  },
```
Correct one was in previous version and it was like "Сохранить в файл" i.e. truncated to first 3 letters now.

**Comment by RealRaven2000** [comment]
> Last [f4e691c](https://github.com/RealRaven2000/FiltaQuilla/commit/f4e691c6534e2b9cc5d18b8a2941f02f1650fb63) update introduces a many errors in the Russian locale, for example now:
> 
> ```
>   "fq.saveMsgAsFile": {
>     "message": "Сох"
>   },
> ```
> 
> Correct one was in previous version and it was like "Сохранить в файл" i.e. truncated to first 3 letters now.

that's weird - we need to check messages.json. I am not aware that I changed anything...
